.gitmodules

```diff
@@ -43,3 +40,12 @@
 [submodule "external/oxen-encoding"]
 	path = external/oxen-encoding
 	url = https://github.com/oxen-io/oxen-encoding.git
+[submodule "external/oxen-logging"]
+	path = external/oxen-logging
+	url = https://github.com/oxen-io/oxen-logging
+[submodule "external/loki-mq"]
```

Comment:
I feel like this is/should be supplanted by the oxen-mq submodule?

Reply:
Yeah thats what i was trying to do. Looks like i didnt delete the loki-mq path correctly.